chromium_src/BUILD.gn

```
@@ -49,6 +66,75 @@ static_library("chrome") {
     "//chrome/browser/ui/views/autofill/autofill_popup_view_utils.h",
     "//extensions/browser/app_window/size_constraints.cc",
     "//extensions/browser/app_window/size_constraints.h",
+
+    "//chrome/browser/webauthn/chrome_authenticator_request_delegate.h",
+    "chrome/browser/webauthn/chrome_authenticator_request_delegate.cc",
```

```suggestion
    "//chrome/browser/webauthn/chrome_authenticator_request_delegate.cc",
```

Seems like this should begin with `//` like the rest?